packages/dx-grid-core/src/plugins/virtual-table-state/helpers.test.ts

```diff
@@ -326,4 +326,35 @@ describe('VirtualTableState helpers', () => {
       });
     });
   });
+
+  describe('#getAvailableRowCount', () => {
+    it('should return totalCount when not infiniteScroll', () => {
```

Comment:
`should return totalCount when not infiniteScroll`
->
`should return totalCount when not infinite scrolling`